cl/aggregation/mocks/AggregationPool.go

```diff
@@ -0,0 +1,65 @@
+// Code generated by mockery v2.42.1. DO NOT EDIT.
+
+package mocks
```

Comment:
files should be aggregation_pool

Reply:
I removed this mock file becasue it's not used for now